client/routes/namespace/workflow-archival/components/archival-disabled-messaging/constants.js

```diff
@@ -6,11 +6,11 @@ export const archivalDisabledMessage = [
 export const historyArchivalDisabledMessage =
   'Set HistoryArchivalState to ENABLED by running command:';
```

Comment:
Consider changing to:
"Run this command to enable Archival for Event Histories:"